pkgs/by-name/bc/bcc/package.nix

```diff
@@ -33,15 +35,25 @@ python3.pkgs.buildPythonApplication rec {
   format = "other";
 
   buildInputs = with llvmPackages; [
-    llvm llvm.dev libclang
-    elfutils luajit netperf iperf
-    flex bash libbpf
+    bash
+    elfutils
+    flex
+    iperf
+    libbpf
+    libclang
+    llvm
+    llvm.dev
+    luajit
+    netperf
   ];
 
   patches = [
     # This is needed until we fix
     # https://github.com/NixOS/nixpkgs/issues/40427
     ./fix-deadlock-detector-import.patch
+    # Quick & dirty fix for bashreadline
+    # https://github.com/NixOS/nixpkgs/issues/328743
+    ./bashreadline.py-remove-dependency-on-elftools.patch
```

Comment:
The file is still missing.

Reply:
Thanks, I couldn't understand the pkgs/by-name check failure without any message attached to it as I was sure I had added the file... (the git add is right here in my shell history, now curious how that went wrong...)

Anyway, fixed. Thanks!